djstripe/decorators.py

```diff
@@ -1,4 +1,12 @@
 # -*- coding: utf-8 -*-
+"""
+.. module:: djstripe.decorators.
+
+   :synopsis: dj-stripe Decorators.
+
+.. moduleauthor:: N/A
```

Comment:
Go ahead and add me, @pydanny, and @audreyr

https://github.com/kavdev/dj-stripe/blame/master/djstripe/decorators.py